### PR TITLE
fix(docz-tools): Fixes the broken search

### PR DESCRIPTION
**Pull request by eddysims**
## Motivations

When we updated the theme, we broke the search functionality.

## Changes

Moves the navigation items into their own component so that the number of hooks called is not changing all the time.

### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- Broken search functionality

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by chris-at-jobber** [comment]
Not sure if this is within the scope of this fix, but if you haven't expanded the menu section for a search result, only the collapsed parent will be visible:
image

**Comment by chris-at-jobber** [comment]
The search no longer breaks Atlantis! ✅

File: packages/docz-tools/src/gatsby-theme-docz/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -12,26 +12,29 @@ interface NavigationProps {
 
 export function Navigation({ query, sidebarRef }: NavigationProps) {
   const items = useJobberMenu(query);
+  const hasQuery = query != undefined && query !== "";
+
+  console.log({ items, query, hasQuery });
```
Comment (seanhealy) [inline packages/docz-tools/src/gatsby-theme-docz/components/Navigation/Navigation.tsx:17]: We missed this.

**Review by seanhealy — CHANGES_REQUESTED** [review]
Just a log to remove then we're golden!

**Review by seanhealy — APPROVED** [review]

**Pull request merged**